modules/@apostrophecms/ui/ui/apos/components/AposCellContextMenu.vue

```diff
@@ -54,11 +56,18 @@ export default {
         classes.push('is-visible');
       }
       return classes;
+    },
+    userId() {
+      return apos.login.user._id;
     }
   },
   methods: {
     handler(action) {
       this.$emit(action);
+    },
+    manuallyPublished(doc) {
```

Comment:
It seems like this would be better as a computed property. Only needs to run once.